checkov/runner_filter.py

```diff
@@ -262,6 +264,10 @@ def within_threshold(self, severity: Severity) -> bool:
         below_max = self.skip_check_threshold and self.skip_check_threshold.level >= severity.level
         return above_min and not below_max
 
+    @staticmethod
+    def secret_check_matches(secret_validation_status: str, statuses_list: list[str]) -> bool:
+        return secret_validation_status.lower() in [status.lower() for status in statuses_list]
```

Comment:
Might be worth having a list of lower case statuses so we don't iterate each time to create the lower case list, then iterate again to find the secret_validation_status, WDYT?

Reply:
When adding the soft/hard fails I changed it to insert the values capitalized if its a secret validation status + added some tests. removed the .lower() from both objects